spectrocloud/data_source_appliances.go

```diff
@@ -0,0 +1,73 @@
+package spectrocloud
+
+import (
+	"context"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/spectrocloud/hapi/models"
+	"github.com/spectrocloud/terraform-provider-spectrocloud/pkg/client"
+	"reflect"
+)
+
+type MyMap map[string]string
+
+func dataSourceAppliances() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourcesApplianceRead,
+
+		Schema: map[string]*schema.Schema{
+			"ids": {
+				Type:     schema.TypeList,
```

Comment:
```suggestion
				Type:     schema.TypeList,
				Description: "The unique ids of the appliances. This is a computed field and is not required to be set.",
```

@nikchern are you aware of a way to avoid generating this field in the docs? It's not needed, and it can cause confusion so if we can omit it that would be best.

Reply:
This field is needed @karl-cardenas-coding